.github/workflows/publish-backend-plugin-manager.yaml

```diff
@@ -50,21 +50,36 @@ jobs:
           else
             GIT_REF=master
           fi
-          echo "GIT_REF=$GIT_REF" >> $GITHUB_ENV
+          echo "GIT_REF=$GIT_REF" >> $GITHUB_OUTPUT
 
-      - name: Checkout
+      - name: Checkout from master
+        if: steps.get_checkout_ref.outputs.GIT_REF == 'master'
         uses: actions/checkout@v4
         with:
           repository: backstage/backstage
-          ref: ${{ steps.get_checkout_ref.outputs.GIT_REF }}
+          ref: master
           # fetch all history, but only for the current ref
           fetch-depth: 2147483647
           fetch-tags: false
 
+      - name: Checkout from a git ref
+        if: steps.get_checkout_ref.outputs.GIT_REF != 'master'
+        uses: actions/checkout@v4
+        with:
+          repository: backstage/backstage
+          ref: '${{ steps.get_checkout_ref.outputs.GIT_REF }}'
+
       - name: Get sha of the latest backend-plugin-manager commit
         id: get_commit_sha
+        env:
+          GIT_REF: ${{ steps.get_checkout_ref.outputs.GIT_REF }}
         run: |
-          echo "COMMIT_ID=$(git rev-list --max-count=1 HEAD -- packages/backend-plugin-manager)" >> "$GITHUB_OUTPUT"
+          echo "GIT_REF=$GIT_REF"
+          if [[ "$GIT_REF" == "master" ]]; then
```

Comment:
you could simply do 
```
if [[ ${{ steps.get_checkout_ref.outputs.GIT_REF }} == "master" ]]; then
```

Reply:
fixed in https://github.com/janus-idp/backstage-plugins/pull/812/commits/fb940dbceedd5e13e2fc2e57e82732d4def4a6c5